<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>391</width>
      <height>311</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Helvetica'; font-size:8pt; font-weight:600; color:#666666;&quot;&gt;Welcome&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Helvetica'; font-size:8pt; color:#666666;&quot;&gt;Dori me Interimo, adapare Dori me Ameno, Ameno Latire Latiremo Dori me Ameno Omenare imperavi ameno Dimere, dimere matiro Matiremo Ameno Omenare imperavi emulari, ameno Omenare imperavi emulari, ameno Ameno dore Ameno dori me Ameno dori me Ameno dom Dori me reo Ameno dori me Ameno dori me Dori me am Ameno Omenare imperavi ameno Dimere dimere matiro Matiremo Ameno Omenare imperavi emulari, ameno Omenare imperavi emulari, ameno Ameno dore Ameno dori me Ameno dori me Ameno dom Dori me reo Ameno dori me Ameno dori me Dori me Ameno dori me Ameno dori me Dori me (dori me, dori me, dori me) (Dori me, dori me, dori me) Ameno Ameno dore Ameno dori me Ameno dori me Ameno dom Dori me reo Ameno dori me Ameno dori me Dori me dom Ameno dore Ameno dori me Ameno dori me Ameno dom Dori me reo Ameno dori me&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionLocal"/>
    <addaction name="actionOwned"/>
   </widget>
   <widget class="QMenu" name="menuUsers">
    <property name="title">
     <string>Users</string>
    </property>
    <addaction name="actionAll"/>
    <addaction name="actionBanned"/>
   </widget>
   <widget class="QMenu" name="menuProfile">
    <property name="title">
     <string>Profile</string>
    </property>
    <addaction name="actionLogin_2"/>
    <addaction name="actionLogout_2"/>
    <addaction name="actionLogout_3"/>
   </widget>
   <addaction name="menuView"/>
   <addaction name="menuUsers"/>
   <addaction name="menuProfile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_from_audio_file">
   <property name="text">
    <string>New from audio file</string>
   </property>
  </action>
  <action name="actionNew_from_real_time">
   <property name="text">
    <string>New from real time</string>
   </property>
  </action>
  <action name="actionLocal">
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionOwned">
   <property name="text">
    <string>Repoted</string>
   </property>
  </action>
  <action name="actionGlobal">
   <property name="text">
    <string>Global</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionRegister">
   <property name="text">
    <string>Register</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionAll">
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionBanned">
   <property name="text">
    <string>Banned</string>
   </property>
  </action>
  <action name="actionLogin_2">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionLogout_2">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionLogout_3">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
